test/unit/imgix-provider.test.jsx

```diff
@@ -17,43 +17,60 @@ describe("ImgixProvider", () => {
     const wrappedComponent = (
       <ImgixProvider>
         <ReactImgix {...imageProps} />
+        <Background {...imageProps} />
       </ImgixProvider>
     );
 
+    const renderedComponent = shallow(wrappedComponent);
+
+    // Inspect the rendered children directly
+    const renderedChildren = renderedComponent.children();
+
     const expectedProps = {
-      children: (
-        <ReactImgix
-          src="https://assets.imgix.net/examples/pione.jpg"
-          sizes="50vw"
-        />
-      ),
       value: {},
     };
 
-    const renderedComponent = shallow(wrappedComponent);
-    expect(renderedComponent.props()).toEqual(expectedProps);
+    expect(renderedChildren.length).toBe(2); // Assuming you have two children
+    expect(renderedChildren.at(0).props()).toEqual({
+      src: 'https://assets.imgix.net/examples/pione.jpg',
+      sizes: '50vw',
+    });
+    expect(renderedChildren.at(1).props()).toEqual({
+      src: 'https://assets.imgix.net/examples/pione.jpg',
+      sizes: '50vw',
+    });
+
+    expect(renderedComponent.prop('value')).toEqual(expectedProps.value);
```

Comment:
suggestion(non-blocking): might make sense to get rid of `expectedProps` since it's only used in this one test now.